lms/envs/common.py

```diff
@@ -1919,10 +1905,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-docstring
         'source_filenames': discussion_vendor_js,
         'output_filename': 'js/discussion_vendor.js',
     },
-    'notes': {
```

Comment:
@dianakhuang just confirming, you're sure this is the JS related to the old notes app and not the JS used by the notes-api service. I know that all the chrome for the notes-api backed notes service is defined in the LMS but I don't know enough to discern it from the old notes JS so I just want to make sure you do?

Reply:
Yup, the JS for the notes-api-service is over here: https://github.com/edx/edx-platform/tree/master/lms/static/js/edxnotes 

It uses Backbone and this one was converted from Coffeescript, which makes it extremely likely that this is unrelated to the working version.